src/style.css

```diff
@@ -0,0 +1,70 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 900;
+}
+
+.bigForm {
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.searchLabel {
+  position: relative;
+}
+
+.serachInput {
+  font-family: Avenir-Book, sans-serif;
+  box-sizing: border-box;
```

Comment:
You can put this style in a separate selector and don't repeat it again in `.searchIcon`
```suggestion
```
`* {
  box-sizing: border-box;
}`